NEWS

```diff
@@ -107,84 +127,114 @@ New Functionality
+  enabled at ``zeek_init()`` time. The set can be populated via
+  :zeek:see:`redef`.  This change only has an effect in settings where
+  ``Analyzer::disable_all`` is changed to ``T``. By default, all analyzers
+  continue to be enabled.
+
+- A new ``analyzer.log`` was added to log all analyzer violations and optionally
+  analyzer confirmations. This log can be useful during development of new
+  analyzers as well as for collecting operational data in production
   environments.
 
+- A new ``join_string_set`` BIF was added, replacing the existing script-level
+  version from utils/strings.zeek.
+
+- A new ``&ordered`` attribute for tables and sets was added. This attribute
+  causes iteration over a table/set to return elements in the order of their
+  insertion.
+
+- A new ``-D`` argument was added to the ``configure`` script to allow passing
+  parameters directly to the underlying CMake call.
+
+- Added parsing for the challenge and response fields to the NTLM analyzer.
+
+- A new ``FTP::max_command_length`` value was added to script-land, defaulting
+  to 100. This value is used by the FTP analyzer to limit the size of commands
+  accepted by the analyzer. A ``FTP_max_command_length_exceeded`` weird is
+  raised for any violations of that length.
+
+- A new ``-O gen-standalone-C++`` argument was added to the ``configure`` script
+  to enable much more robust support for compiling Zeek scripts to
+  self-contained C++.
+
+- A new ``-O allow-cond`` argument was added to the ``configure`` script to
+  enable compiling scripts to C++ that are influenced by ``@if`` conditionals.
+
+- Added built-in functions to speed up cat() BiF calls when using "-O ZAM".
+
 Changed Functionality
 ---------------------
 
-- Violations for packet analyzers that have sessions attached with them
-  will be raised once only. Further, analyzer confirmations are not raised
-  after a violation.
+- Violations for packet analyzers that have sessions attached with them will be
+  raised once only. Further, analyzer confirmations are not raised after a
+  violation.
 
-- The parameter given to ``enum_names()`` can now be a string naming the
-  enum type, rather than the type itself.
+- The parameter given to ``enum_names()`` can now be a string naming the enum
+  type, rather than the type itself.
 
-- The ``type_name`` of enum types produced by ``record_fields()`` now
-  includes the actual type name rather than just ``"enum"``.
+- The ``type_name`` of enum types produced by ``record_fields()`` now includes
+  the actual type name rather than just ``"enum"``.
 
 - Passing non-string ``sep`` and ``def`` arguments to ``cat_sep()`` isn't a
   fatal error anymore. More descriptive error messages are produced, too.
 
 - The number of analyzer violation events that can be raised by protocol
-  analyzer instances is now capped by the const ``max_analyzer_violation_events``.
+  analyzer instances is now capped by the const
+  ``max_analyzer_violation_events``.
```

Comment:
Uhm, there might have been a conflict resolution hickup on my side. This is the same as below, but without mentioning file analyzers which came as a follow up.

Can you drop this hunk in favor of the one below?

Reply:
Done